charts/coredns/experiments.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Deletes coredns pod in kube-system namespace
+kind: ChaosExperiment
+metadata:
+  name: coredns-pod-delete
+  version: 0.1.0
+spec:
+  definition:
+    permissions:
+      apiGroups:
+        - ""
+        - "extensions"
+        - "apps"
+        - "batch"
+        - "litmuschaos.io"
+      resources:
```

Comment:
can we limit these permissions to the ones we really need?

Reply:
done